packages/data/src/plugins/persistence/test/index.js

```diff
@@ -384,13 +384,13 @@ describe( 'persistence', () => {
 	} );
 } );
 
-describe( 'migrateFeaturePreferencesToInterfaceStore', () => {
```

Comment:
To most easily review this, toggle on 'Hide whitespace' in the little cog menu at the top.

Otherwise the diff looks worse than it is. Lots of the other tests were all incorrectly nested in this `describe`, and all I did was:
- un-nest them
- delete the tests for the removed function
- add the tests for the new function.